cypress/Contentful/ContentfulApi.js

```diff
@@ -22,12 +22,12 @@ class ResponseWrapper {
 }
 
 export class ContentfulApi {
-  static getEntryCollection(query) {
+  static getEntryCollection(query, failOnStatusCode=true) {
```

Comment:
It's not super clear what the purpose of `failOnStatusCode` is. Possible to add something to make it more clear or a comment? I see it's reused a lot so that might be a challenge

Reply:
`failOnStatusCode` is the name Cypress uses - I was just copying them. Luke asked about this too though, so I'll think of a better name and add some comments.